<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TEA Encoder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonEncrypt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Encrypted values</string>
    </property>
    <widget class="QLineEdit" name="lineEditEncryptedV1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEncryptedV2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelEncryptedV1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>V1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelEncryptedV2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>V2:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRawValues">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>381</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Raw values</string>
    </property>
    <widget class="QLabel" name="labelV1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>V1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelV2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>V2:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditV2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditV1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelK1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>K1:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditK1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelK2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>K2:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditK2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditK3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelK3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>K3:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditK4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelK4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>K4:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
